### ISSUE TITLE
Allow archived chats to be put into a specific folder

### ISSUE THREAD

**Issue by SurDno** [telegramdesktop/tdesktop#7906]

**Comment by RememberTheAir** [comment]
You can include your archived chats in any folder - maybe you wanted to ask for something like 6th point of #7500?

**Comment by SurDno** [comment]
Yes, I can put any private conversations, group or channels in the new folder one by one, but I can't put archived chats as a whole in there. It'd be very convenient to do so, because that's the only thing stopping me that still forces me to use 'All chats'.

**Comment by KlausDevWalker** [comment]
> Yes, I can put any private conversations, group or channels in the new folder one by one, but I can't put archived chats as a whole in there. It'd be very convenient to do so, because that's the only thing stopping me that still forces me to use 'All chats'.

I support you, buddy. We know that's possible, cause for example Telegraph has "all admin" folder. Just it's not synced cause it's client side. My two cents is that Telegram could let us use "excluded" filters as "included" ones. So we'll have something like "all muted" without having to archive chats.

**Comment by KlausDevWalker** [comment]
Of course not!. This issue wasn't addressed yet.

**Comment by FrozenFruitsFrenzy** [comment]
Of course not!. This issue wasn't addressed yet.
